mcode/mcode-coverage-checker
# Adding Disease coverage checking

# Description
Issue: STEAM-843

This PR adds coverage checking for Disease resources. 

## Important Changes
Coverage checking has been added for the disease section of the mCODE diagram in `diseaseCoverage.js`. The TNM category resources didn't have any fields listed on the diagram so I did my best to pick a representative field to check for, let me know if that is something we wanted checked or if any additional fields should be checked.

Tests added for Disease coverage checking in `diseaseCoverage.test.js` and accompanying `diseaseBundle.json` test bundle created.

## Testing Recommendations
- Ensure that new tests cover sufficient cases and pass
- Check that appropriate fields are checked for
- Make sure that each field has a proper true/false value when it is covered or not

# Checklists

**Submitter:**

@ACCT1 :
